### PR TITLE
Update ui_code_a_life_bar.rst

### PR DESCRIPTION
**Pull request by corrigentia**

### CODE REVIEW

File: getting_started/step_by_step/ui_code_a_life_bar.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ The scene contains a background sprite, a GUI, and two characters.
 
    The scene tree, with the GUI scene set to display its children
 
-The GUI scene encapsulates all of the Game User Interface. It comes with
+The GUI scene encapsulates all of the game's Graphical User Interface. It comes with
```
Comment (bojidar-bg) [inline getting_started/step_by_step/ui_code_a_life_bar.rst]: As mentioned in the other PR, GUI stands for Graphical User Interface.

File: getting_started/step_by_step/ui_code_a_life_bar.rst [hunk 2]
```diff
@@ -556,7 +556,7 @@ Try the game again to see a nice blocky animation.
 
 .. figure:: img/lifebar_tutorial_number_animation_working.gif
 
-   By rounding out animated\_health we hit two birds with one stone
+   By rounding out animated\_health, we killed two birds with one stone
```
Comment (bojidar-bg) [inline getting_started/step_by_step/ui_code_a_life_bar.rst]: Maybe keep it in present tense: `kill`

Comment (akien-mga) [reply getting_started/step_by_step/ui_code_a_life_bar.rst]: Also sounds better to me in present tense, I'll add that fixup.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**